src/syncDataSilos.ts

```diff
@@ -188,6 +188,8 @@ export interface DataSiloEnriched {
   identifiers: {
     /** Name of identifier */
     name: string;
+    /** True if identifier is wired */
+    isConnected: boolean;
```

Comment:
What exactly is a wired/connected identifier?

Reply:
<img width="720" alt="Screen Shot 2022-03-28 at 2 14 00 PM" src="https://user-images.githubusercontent.com/10264973/160488256-92b236d8-fda1-41d5-9e2d-84a07aeabc85.png">

the API returns back all possible identifiers that can be enabled to disabled based on the type of integration (i.e. for twilio, its not possible to wire up an email)

i thought the API was returning connected identifiers instead of all possible identifiers